bin/parser/index.js

```diff
@@ -15,25 +13,42 @@ function parseChangelogItem(item) {
   };
 }
 
-function isUsableItem(potentialItem) {
-  return potentialItem.split('\n').length > 1;
+const BADLY_FORMATTED_CHANGELOG = `Your CHANGELOG.md seems to be badly formatted.
+Every item should start with #v1.0.0\n##Release title\n`;
+
+function parseChangelog(stdOut) {
+  try {
+    return getItemsFromStdOut(stdOut);
+  } catch (e) {
+    console.error(e.message);
+    throw new Error(BADLY_FORMATTED_CHANGELOG);
+  }
 }
 
-const BADLY_FORMATTED_CHANGELOG = `An item in your CHANGELOG seems to be badly
-formatted. Please check you have no trailing separator at the end of your CHANGELOG`;
+function getItemsFromStdOut(stdOut) {
+  let indexes = [];
```

Comment:
I find this function hard to understand (😳), could you extract it a bit to make it more understandable?

Should we throw bad format error if any of the older items are incorrectly formatted?

Why not split it to items with Regex (using `.split`) and then check the items?